docs/builtins/General/compose.md

```diff
@@ -3,7 +3,7 @@
 Use `compose` to compose functions where `oper1` performs an operation using the specified `value` and `oper2` takes the results from `oper1` as input to produce the result for the composed function.
 
 You can use any data type for the `value` argument as long as the first `oper1` functions can take that same data type.
-By convention, the data type <a> is used to represent a type-bound parameter like the `value` argument in this function.
+By convention, the data type ```<a>``` is used to represent a type-bound parameter like the `value` argument in this function.
```

Comment:
you can just use a single ` around the thing. Triple ticks are usually for multi-line stuff, although this is fine as well.